undertow/src/main/java/org/wildfly/extension/undertow/ListenerAdd.java

```diff
@@ -73,6 +78,20 @@ public HttpHandler wrap(HttpHandler handler) {
                 }
             });
         }
+        String disallowedMethods = ListenerResourceDefinition.DISALLOWED_METHODS.resolveModelAttribute(context, model).asString();
```

Comment:
you should use something like
final List<String> disallowedMethods = ListenerResourceDefinition.DISALLOWED_METHODS.unwrap(context, model);